client/src/plugins/element-templates-modal/ElementTemplatesModal.js

```diff
@@ -44,7 +44,14 @@ export default class ElementTemplatesModal extends PureComponent {
     this.setState({ activeTab });
   }
 
-  handleBpmnModelerConfigure = async ({ middlewares }) => {
+  handleBpmnModelerConfigure = async ({ middlewares, tab }) => {
+
+    // TODO: Niklas to remove
+    // temporary workaround to deal with the co-dependecies between C8 elementTemplates and plugin

```

Comment:
```suggestion
    // todo(Skaiir): workaround to deal with the co-dependencies between cloud element templates and plugins
    // cf. https://github.com/camunda/camunda-modeler/issues/2731
```

Reply:
Just a proposal 😅

Reply:
As a good practice, please `todo(GITHUB_HANDLE): something` in the way Niklas suggested.

Why: Because we want to follow up when we stuble across the todo in five years and do not recall our names anymore. :wink: